sentences/DPT-surface.js

```diff
@@ -0,0 +1,26 @@
+/**
+Depth:
+$IIDPT,x.x,x.x,,*hh
+ I I_Sensor offset, >0 = surface transducer distance, >0 = keel transducer distance.
```

Comment:
I think we should have better, correct comments or no comment here and refer to some other doumentation.

Reply:
I could add a condensed version of the comments (from the link in PR) to the file:
[DPT - Depth of Water](https://gpsd.gitlab.io/gpsd/NMEA.html#_dpt_depth_of_water)

        1   2   3   4
        |   |   |   |
 $--DPT,x.x,x.x,x.x*hh<CR><LF>

Field Number:
    - Water depth relative to transducer, meters
   -  Offset from transducer, meters positive means distance from transducer to water line negative means distance from transducer to keel
   - Maximum range scale in use (NMEA 3.0 and above)
   - Checksum